script.js

```diff
@@ -1,6 +1,27 @@
 /* eslint-disable linebreak-style */
+/* eslint-disable prefer-destructuring */
+/* eslint-disable no-loop-func */
+
 const hamburger = document.querySelector('.hamburger');
 const toolbar = document.querySelector('.mobile-menu');
+const form = document.getElementById('contact');
+const errorMsg = document.getElementById('text');
+const inputName = document.getElementById('name');
+const inputEmail = document.getElementById('email');
+const inputMsg = document.getElementById('msg');
+
+const projectCard = [
+  {
+    name: 'Multi Post Stories',
```

Comment:
Pop-window does not display anything on mobile and desktop instead it just display a grey overlay. KIndly add project with details on the pop-up. 
![mobile](https://user-images.githubusercontent.com/35504149/136651099-f406adb1-87da-4de1-8109-7dd76864fb4b.png)
![desk](https://user-images.githubusercontent.com/35504149/136651109-367ddc26-16dd-47cc-b06d-2102f4aeb5de.png)